collector.c

```diff
@@ -350,57 +266,66 @@ pgws_collector_main(Datum main_arg)
 	 */
 	pqsignal(SIGTERM, handle_sigterm);
```

Comment:
Maybe we can use SignalHandlerForShutdownRequest here?

Reply:
This doesn't regard to my patch explicitly. But as I've already started big refactoring in current work I'll add this predefined callback to patch. But it was added to pg starting from 14 version.